usr/lib/systemd/system/containerbuild-regionsrv.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Provides Cloud SMT info and credentials to container build tools
+After=network-online.target guestregister.service
+Wants=network-online.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/containerbuild-regionsrv
```

Comment:
Typo? s/=-/=

Reply:
No. The `-` means that the file is optional and systemd won't complain when it is not present.

Reply:
Ah, ack.